scripts/Dockerfile

```diff
@@ -7,6 +7,9 @@ WORKDIR /build
 
 # Adding and installing packages
 ADD package.json yarn.lock ./
+
+RUN mkdir -p /build/scripts
+ADD ./scripts/postinstall.js /build/scripts/
```

Comment:
That's what I was missing on docker when I tried to export my script out of package.json. Good to know 👍

Reply:
Long term I'd like scripts to only contain things that pertain to developers/building and move all ci stuff to a separate folder (or maybe to `scripts/ci`) idk. anyway.